common/turbo_exec/TE2.0_register.tpt2

```diff
@@ -28,35 +28,35 @@
 :global int turbo.cycles
 
 ; This variable determines how many cycles turbo is allowed to run for at most
-; on this frame, it is being reset each frame. If your script requries turbo to
-; run for longer then 200 cycles, you need to set this using this variable.
+; on this frame. It is reset each frame. If your script needs turbo to run for
+; longer then 200 cycles per frame, you need to set that using this variable.
 ; To prevent conflicts with other scripts, please make sure to use the
 ; following instruction to require more cycles :
 ; turbo.cycles.max = max(turbo.cycles.max, turbo.cycles + cycles_you_require)
```

Comment:
Is there a race condition with setting this, or is it always guaranteed that this will be set before a client script has a change to set it, even if it's the first thing that happens after executesync("turbo start")?

If it's safe, you might want to document the reason why.